docs/index.rst

```diff
@@ -43,12 +43,18 @@ The problems that Pipenv seeks to solve are multi-faceted:
 Install Pipenv Today!
 ---------------------
 
+Pipenv is a python package and so can be installed using ``pip`` as you would expect.
 ::
 
     $ pip install pipenv
     ✨🍰✨
 
-If you have excellent taste, there's also a  `fancy installation method <https://docs.pipenv.org/install.html#fancy-installation-of-pipenv>`_.
+However, installing pipenv using pip should be considered a last resort. To
```

Comment:
I'm not sure we want this wording. Pip is a fine option for installing pipenv and is the common case. I don't think we want to push users away from that because it provides an additional barrier for entry.